src/news/ftb-removal.md

```diff
@@ -0,0 +1,45 @@
+---
+title: FTB Modpack Downloading Removal.
+description: Why FTB Modpack Downloading was Removed.
+date: 2023-04-27
+tags:
+  - News
+---
+
+At the request of Slowpoke, representing the FTB team, we have removed the feature that allows downloading FTB modpacks from Prism Launcher. [Github PR][pr-ftb-removal]
+
+Existing instances of FTB modpacks will remain functional, but once the PR is merged, downloading FTB packs directly will no longer be possible.
```

Comment:
Muggles don't know what merging a PR is. Say "with the next release" instead.

Reply:
Haha ok I'll do it when I'm home

Reply:
> Muggles don't know what merging a PR is. Say "with the next release" instead.

That feature isn't even developed yet, I think it's too early to say "with the next release"